apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js

```diff
@@ -133,11 +133,9 @@ async function setClientIfContactPhoneAndTicketAddressMatchesResidentFields (ope
 
     if (residentUserId) {
         const residentUser = await getById('User', residentUserId)
-
-        setClientNamePhoneEmailFieldsByDataFromUser(residentUser, resolvedData)
```

Comment:
please! write a test if resident tryes to create two Resident with the same address/unit

Reply:
we have such test in `Resident.test.js` on line 61